src/tests/cypress/integration/service_transactions_statistics.spec.e2e.js

```diff
@@ -0,0 +1,53 @@
+/* global cy */
+describe('Gateway account statistics page', () => {
+  it('Renders today, week, month statistics successfully', () => {
+    cy.visit('/')
+
+    cy.get('a[href="/services"]').click()
+    
+    cy.contains('Mock').click()
+
+    cy.get('a[href="/gateway_accounts/1"]').click()
+    cy.get('.govuk-button').contains('View statistics').click()
+    cy.get('a[href="/transactions/statistics/?period=today&account=1"]')
+      .should('have.class', 'transactions-filter__item')
+      .and('have.class', 'selected')
+      .and('have.class', 'no-decoration')
+    cy.get('a[href="/transactions/statistics/?period=week&account=1"]')
+      .should('have.class', 'transactions-filter__item')
```

Comment:
There shouldn't be any need to test the CSS/ templating here, I think asserting the `selected` being applied to the appropriate item should be enough!